test/native/integration/blocks-raw-handling.native.js

```diff
@@ -0,0 +1,587 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import path from 'path';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createBlock,
+	getBlockContent,
+	pasteHandler,
+	rawHandler,
+	registerBlockType,
+	serialize,
+} from '@wordpress/blocks';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+function readFile( filePath ) {
+	return fs.existsSync( filePath )
+		? fs.readFileSync( filePath, 'utf8' ).trim()
+		: '';
+}
+
+// Path to the fixtures provided in `gutenberg/test/integration`.
+const fixturesPath = `${ __dirname }/../../integration`;
+
+// NOTE: This file is a clone of the same `blocks-raw-handling.js` file located in
+// `gutenberg/test/integration`. The reason for the separation is that several of
+// the test cases fail in the native version. For now, we are going to skip them, but
+// we'd need to work on them in the future.
+//
+// Once all issues in tests are addressed, we'll remove this file in favor of the
+// original one.
+describe( 'Blocks raw handling', () => {
```

Comment:
As explained in the inline comment, this file is a copy with an extra test case. In a follow-up GitHub issue, we'll track the different test cases to address.

Reply:
> Unless I'm misunderstanding why the snapshots are obsolete in the first place, I found that several of the skipped tests only needed the snapshots updated in the native context to pass. I've added those in https://github.com/WordPress/gutenberg/commit/ca8cf355cb4a699c2f6af6db1eec6fddc67ee3e8 as an experiment. If they look promising, feel free to incorporate in this PR, or we could use it as the starting point for a followup PR.

I originally updated the snapshots in https://github.com/WordPress/gutenberg/pull/56723/commits/78dab049633e0da2303709189486e9ed458c21de for the native version, however, I was concerned that the output might not be the expected one compared to the web snapshots. That's why I updated it in https://github.com/WordPress/gutenberg/pull/56723/commits/bce611a63c812587e1fb250af46586c992b1bb50. This resulted in disabling other test cases referenced in https://github.com/WordPress/gutenberg/pull/56723/commits/5e44132602e636e4cc7a8801dc8074129a5461f9. It might be possible that the output of the processing differs in the native version, but we'd need to check first that the differences are (1) expected and (2) won't lead to conflicts in the content between platforms.

That said, I'd postpone updating the snapshots until we investigate further what's causing the test failures.